.golangci.yml

```diff
@@ -90,14 +90,6 @@ linters:
         # See https://github.com/open-telemetry/opentelemetry-collector/issues/2789
         - fieldalignment
       enable-all: true
-      # settings per analyzer
-      settings:
-        printf: # analyzer name, run `go tool vet help` to see all analyzers
-          funcs: # run `go tool vet help printf` to see available settings for `printf` analyzer
-            - (github.com/golangci/golangci-lint/v2/pkg/logutils.Log).Infof
-            - (github.com/golangci/golangci-lint/v2/pkg/logutils.Log).Warnf
-            - (github.com/golangci/golangci-lint/v2/pkg/logutils.Log).Errorf
-            - (github.com/golangci/golangci-lint/v2/pkg/logutils.Log).Fatalf
```

Comment:
This looks like valid configuration for the printf analyzer (looking at the example https://golangci-lint.run/usage/linters/#govet), is it being enforced somewhere else?

Reply:
thanks for fixing!